### ISSUE TITLE
Building the @excalidraw/excalidraw package causes the example app to stop working

### ISSUE THREAD

**Issue by AriTheElk** [excalidraw/excalidraw#8871]
Not really sure what's going on; I'm trying to fork excalidraw to add collaboration features, but I'm seeing a lot of weird behavior.

Firstly, this is something I can reproduce 100% of the time:

```
git clone https://github.com/excalidraw/excalidraw.git
cd excalidraw
yarn
yarn start
```

Everything works, but if I run:

```
yarn workspace @excalidraw/excalidraw build:esm
yarn start
```

I see this:

![Image](https://github.com/user-attachments/assets/b916c5cf-cc30-44c8-a05d-92ec01adf12f)

Any idea whats going on here?

I also am seeing weird behavior when attempting to use the latest `master` branch in my project via `yarn link`, the app is basically an unrecognizable mess. **Is the current master branch stable?** I've tried also specifically checking out the latest @next release `0.17.1-c0b80a0` and still seeing the same thing.

**Comment by Mrazator** [comment]
> Any idea whats going on here?

Yeah, `excalidraw-app` does not work on top of built excalidraw package with esbuild, instead it builds the whole thing from scratch with vite (rollup). We might change it in the future.

> I also am seeing weird behavior when attempting to use the latest master branch in my project via yarn link, the app is basically an unrecognizable mess.

What kind of weird behavior are you encountering?

> Is the current master branch stable?

Master isn't stable, release/ is what is running at excalidraw.com. Past releases are stable in terms of public API.

**Comment by Mrazator** [comment]
Also, there might be issues with `yarn link` specifically, consider either creating a yarn workspace or "hard linking" the path with `"@excalidraw/excalidraw": "../your/relative/path"`.
